src/Angular.js

```diff
@@ -125,6 +128,39 @@ var VALIDITY_STATE_PROPERTY = 'validity';
 
 var hasOwnProperty = Object.prototype.hasOwnProperty;
 
+var objectMaxDepthInErrorMessage = 5;
+
+/**
+ * @ngdoc function
+ * @name angular.configureErrorHandling
+ * @module ng
+ * @kind function
+ *
+ * @description Use this function to configure error handling.
+ *
+ * @param {Object=} config An object for defining configuration for error handling. The
+ *     following keys are supported:
+ *
+ * * `objectMaxDepth` - used to stringify objects in error messages until reaching the max depth.
+ * if it is not supplied the default max depth which is 5 will be used instead, or if it is invalid
+ * (less than `1` or not a `Number`) no max depth will be used.
+ *
+ */
+function configureErrorHandling(config) {
+  if (isObject(config)) {
+    objectMaxDepthInErrorMessage = isValidObjectMaxDepth(config.objectMaxDepth) ? config.objectMaxDepth : NaN;
+  }
+}
```

Comment:
Did you mean !config in ??

``` javascript
if (config) {
    return {
      objectMaxDepth: objectMaxDepthInErrorMessage
    };
  } 
```

you want to return configuration in case of config arg is not supplied ?